android/README.md

```diff
@@ -1,8 +1,15 @@
 # TradingView Charting Library Android Integration Example
 
+The earliest supported version of the charting library for these examples is `v20.033`.
+
 ## How to start
 
-1. Copy all files from https://github.com/tradingview/charting_library/ to `app/src/main/assets/charting_library/`. The earliest supported version of the Charting Library is 17. If you get 404 then you need to [request an access to this repository](https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/).
+1. Check that you can view https://github.com/tradingview/charting_library/. If you do not have access then you can [request access to this repository here](https://www.tradingview.com/HTML5-stock-forex-bitcoin-charting-library/).
+1. Install dependencies `npm install`.
+1. Copy the charting library files
+	1. If you are able to run bash scripts then the `copy_charting_library_files.sh` script can be used to copy the current stable version's files.
+	1. If you are not able to run bash scripts then do the following:
```

Comment:
Isn't there a way to only put that bit of text somewhere "central" and link it to all README files rather than mainly copying/pasting?
I get that examples may not change very often but we could later imagine having a script of some sort checking if at least the examples still work with vXYZ of CL without having to amend a bunch of files just to reflect a version change.

Reply:
We could extract the common text to the root README and link to it from the sub-READMEs?